scss/_forms.scss

```diff
@@ -179,10 +179,11 @@
   position: relative;
   display: block;
   // margin-top:    ($spacer * .75);
-  margin-bottom: ($spacer * .75);
+  margin-bottom: ($spacer * .75) - $input-padding-y;
 
   label {
     padding-left: 1.25rem;
+    padding-top: $input-padding-y;
```

Comment:
`scss/_forms.scss:185 [W] PropertySortOrder: Properties should be ordered padding-top, padding-left, margin-bottom, font-weight, cursor`